js/src/scrollspy.js

```diff
@@ -212,6 +217,7 @@ const ScrollSpy = (($) => {
         if (this._activeTarget !== target) {
           this._activate(target)
         }
+        return
```

Comment:
Ah, sorry, I was on mobile and it looked like this was already the end of the function (obviously it's not).